src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -1017,15 +1017,16 @@ public function testColumnStyle()
 
     public function testThrowsWhenTheCellInAnArray()
     {
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage('A cell must be a TableCell, a scalar or an object implementing "__toString()", "array" given.');
-        $table = new Table($output = $this->getOutputStream());
+        $table = new Table($this->getOutputStream());
         $table
             ->setHeaders(['ISBN', 'Title', 'Author', 'Price'])
             ->setRows([
                 ['99921-58-10-7', [], 'Dante Alighieri', '9.95'],
             ]);
 
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('A cell must be a TableCell, a scalar or an object implementing "__toString()", "array" given.');
+
         $table->render();
```

Comment:
@stof it seems, that this is no dead code, as the exception is raised when calling `render()` and not when calling `setRows()` 👍

Reply:
then in the other PR, keep `setRows` in initialization